LongWarOfTheChosen/Src/LW_Overhaul/Classes/CHItemSlot_PistolSlot_LW.uc

```diff
@@ -59,9 +60,9 @@ static function bool CanAddItemToPistolSlot(
     WeaponTemplate = X2WeaponTemplate(Template);
     if (WeaponTemplate != none)
     {
-        return WeaponTemplate.WeaponCat == 'pistol';
+        return default.PISTOL_SLOT_WEAPON_CLASSES.Find(WeaponTemplate.WeaponCat()) == INDEX_NONE;
```

Comment:
Shouldn't it be != INDEX_NONE?

Reply:
Could be? Would != INDEX_NONE mean that "if it´s not on the index, do this"? I thought it would be the other way around but now I am not so sure. 
Anyhow, will revert these and open a new branch (not familiar with Github at all so I thought it would do it automatically on a new PR)

Reply:
INDEX_NONE means that the thing has not been found in an array.